versionist.conf.js

```diff
@@ -1,32 +1,62 @@
-module.exports = {
+var _ = require('lodash');
 
-  subjectParser: 'angular',
+lowerKeys = (obj) => {
+  return _.transform(obj, (result, val, key) => {
+    result[key.toLowerCase()] = val;
+  });
+};
 
+module.exports = {
+  // This setup allows the editing and parsing of footer tags to get version and type information,
+  // as well as ensuring tags of the type 'v<major>.<minor>.<patch>' are used.
+  // It increments in a semver compatible fashion and allows the updating of NPM package info.
+  editChangelog: true,
+  parseFooterTags: true,
   getGitReferenceFromVersion: 'v-prefix',
+  incrementVersion: 'semver',
+  updateVersion: 'npm',
 
+  // Always add the entry to the top of the Changelog, below the header.
   addEntryToChangelog: {
     preset: 'prepend',
     fromLine: 5
   },
 
-  updateVersion: 'npm',
-
+  // Only include a commit when there is a footer of 'change-type'.
+  // Ensures commits which do not up versions are not included.
+  // Commit messages without a relevant footer will not be included in the CHANGELOG.
   includeCommitWhen: (commit) => {
-    return commit.footer['Changelog-Entry'];
+    commit.footer = lowerKeys(commit.footer);
```

Comment:
Unfortunately, I don't agree. From previous experience, it can be easy to misremember the format of a tag; having to then go back and edit these, when the actual tag doesn't make it into a Changelog or `package.json` entry makes no sense to me.

I would rather the 'stricter' option be opt-in than opt-out.